CanvasCore/CanvasCore/ReactNativeModules/NativeAccessibility.m

```diff
@@ -43,6 +43,7 @@ - (nullable UIView *)subviewWithIdentifier:(NSString *)identifier {
 
     NSMutableArray<UIWindow *> *windows = [NSMutableArray new];
     NSArray<UIScene *> *connectedScenes = [[UIApplication.sharedApplication connectedScenes] mutableCopy];
```

Comment:
`connectedScenes` returns an `NSSet` so we should use that type instead of `NSArray` or convert the set to an array using the `allItems` property. `connectedScenes` variable doesn't seem to be mutated so the `mutableCopy` call can be removed.